localization/vi/abstract-document/README.md

```diff
@@ -0,0 +1,188 @@
+---
+layout: pattern
+title: Abstract Document
+folder: abstract-document
+permalink: /patterns/abstract-document/
+categories: Structural
+language: vi
+tags: 
+ - Extensibility
+---
```

Comment:
Please check the YAML frontmatter requirements in https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute

Reply:
I don't know if I understand your requirement right. But I did remove `layout`, `folder`, `permalink`, and change `categories` to `category` already